### ISSUE TITLE
<search> element

### ISSUE THREAD

**Issue by domenic** [mozilla/standards-positions#610]
## Request for Mozilla Position on an Emerging Web Specification

* Specification Title: `<search>` element
* Specification or proposal URL: https://github.com/whatwg/html/pull/7320
* Mozillians who can provide input (optional): @annevk, @smaug----, and @hsivonen have been involved in these discussions

**Comment by annevk** [comment]
From @hsivonen in a comment on the PR:

> If a `<search>` element is added, making it close `<p>` is probably the right thing to do. I'm not convinced that adding the `<search>` element is worthwhile, since it doesn't add expressiveness but mainly addresses a principled concern that all of ARIA should be ported to HTML in non-ARIA form, but I don't know enough about the use cases to argue strongly in favor or against.

I don't have a particularly strong opinion on this either, though I do think it's valuable for HTML to be expressive enough that in most situations web developers don't have to reach for ARIA.

Overall I'd lean towards worth prototyping for this therefore.

**Comment by bgrins** [comment]
We're generally supportive of the goal here, and when I initially looked at the data I found [top 10 most common aria roles on HTTP archive](https://almanac.httparchive.org/en/2021/accessibility#aria-roles) I found this element really compelling (~18% of pages have a role=search and it's the most common role other than `presentation` which doesn't have an element). 

However, when digging deeper into the data I believe that `form role=search` is much more common than `div role=search`. From a preliminary analysis on HTTP Archive that @rviscomi was kind enough to help with it appears that for the search role, ~16% of pages use form and ~2% use div. I'm not saying I'm opposed to the new element, but that does seem to limit the benefit as compared to i.e. `nav` which doesn't have this split behavior with forms. And, could there be confusion with pages currently using the form with role knowing whether or not they should be migrating from `<form role=search>` to `<search><form>` or `<form><search>`? @domenic I didn't see discussion about this in the issue but may be missing it - has this already been considered?

**Comment by domenic** [comment]
Yeah, that was discussed around https://github.com/whatwg/html/issues/5811#issuecomment-956454034 and subsequent comments. Apparent either `<search><form>` or `<form><search>` are exposed the same way to AT. We have an example in the spec draft of the former.

**Comment by zcorpan** [comment]
> Apparent either `<search><form>` or `<form><search>` are exposed the same way to AT.

@jcsteh commented on this point in https://github.com/whatwg/html/issues/5811#issuecomment-1159161401 , but the conclusion still seems to be that the experience for AT users is probably the same. I don't see any impact on the spec PR for this point, though, but can keep in mind when explaining or documenting the element.

I've discussed with @hsivonen and @jcsteh; we're not completely sold on the idea that all ARIA roles must have an HTML equivalent, but also not opposed.

["worth prototyping" is now called "positive"](https://github.com/mozilla/standards-positions/pull/680). I'll add the label; no dashboard entry (unless requested).
